### DISCUSSION TITLE
Does the hobby plan usage limits resets monthly?

### DISCUSSION THREAD

**Discussion by Ronuhz** [vercel/community#3130] [Feature Question] [answered]
### Question

Hello
My question is what's in the title. **Does the hobby plan usage limits resets monthly?**. Like for example I have 100 GB of Bandwidth for each month, and same for the other features included in it.

**Comment by amyegan** [comment, marked as answer]
The Hobby plan usage limits are reset on a rolling 30 day period. So it's always looking at the past 30 days.

**Reply by Ronuhz** [reply]
Okay thanks for the clarification.
